app/assets/javascripts/omni-nav-v2.js

```diff
@@ -186,7 +186,14 @@ var OmniNav2 = (function() {
       var idSelector = '#' + anchorIdentifer;
 
       // Look first for name selector.
-      var $target = $(nameSelector);
+      // anchorIdentifier could lead to an invalid jQuery selector so wrap in a
+      // try/catch. For details, see https://github.com/chapmanu/cascade-assets/issues/295.
+      try {
+        var $target = $(nameSelector);
+      }
+      catch(error) {
+        return false;
+      }
```

Comment:
Here's the fix. This should keep our code from crashing the javascript on the page. 

I'll deploy to Cascade Dev and test on the page Sarah identified in the issue.